<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zevin</class>
 <widget class="QWidget" name="zevin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>金泽文_decode</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/zevin/Desktop/QKJEYKFQZY}{@VHETCMKH`M.png</normaloff>C:/Users/zevin/Desktop/QKJEYKFQZY}{@VHETCMKH`M.png</iconset>
  </property>
  <property name="locale">
   <locale language="Chinese" country="China"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>801</width>
     <height>291</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Chinese" country="China"/>
   </property>
   <property name="title">
    <string>Decode</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="Chinese" country="China"/>
    </property>
    <property name="text">
     <string>http link: 
or drag the 
picture here:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>180</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="Chinese" country="China"/>
    </property>
    <property name="text">
     <string>Decode_local_file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>40</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="Chinese" country="China"/>
    </property>
    <property name="text">
     <string>Decode_network</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="Chinese" country="China"/>
    </property>
    <property name="text">
     <string>local path:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="Chinese" country="China"/>
    </property>
    <property name="text">
     <string>./QRCode.bmp</string>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>9</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="locale">
     <locale language="Chinese" country="China"/>
    </property>
    <property name="text">
     <string>http://img3.epanshi.com/7526/upload/%E4%B8%AD%E5%8D%97%E5%B9%95%E5%A2%99%E4%BA%8C%E7%BB%B4%E7%A0%81.png</string>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>801</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Output</string>
   </property>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="Chinese" country="China"/>
    </property>
   </widget>
   <zorder>textBrowser</zorder>
   <zorder>pushButton</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>label_4</zorder>
   <zorder>pushButton</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>label_4</zorder>
  </widget>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
